commands/web-searches/postgresql.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title PostgreSQL Documentation
+# @raycast.mode silent
+#
+# Optional parameters:
+# @raycast.packageName Web Searches
+# @raycast.icon images/psql.png
+# @raycast.argument1 { "type": "text", "placeholder": "Query" }
+#
+# Documentation:
+# @raycast.author Phil Salant
+# @raycast.authorURL https://github.com/PSalant726
+# @raycast.description Search PostgreSQL documentation
+
+open "https://www.postgresql.org/search/?q=${1// /+}"
```

Comment:
I made a quick test against PostgreSQL and they accept `%20` for spaces as well, then we can use the `percentEncoded` instead of the manual replacement.
```suggestion
open "https://www.postgresql.org/search/?q=$1"
```